File: C/unit_5/Bc.c
#include <stdio.h>
int CalcPow(int x, int y)
{
    int i;
    int n = 1;
    ;
    for (i = 0; i < y; ++i)
        n *= x;
    return (n);
}
int main()
{
    int n, e, a;
    printf("Enter the Number : \n");
    scanf("%d", &n);
    printf("Enter the Exponent :\n");
    scanf("%d", &e);
    a = CalcPow(n, e);
    printf("The final Answer is : %d \n", a);
    return 0;
}

File: Java/unit_5/Bc.java
package Java.unit_5;

import java.util.Scanner;

public class Bc {
  static int CalcPow(int x, int y) {
    int i;
    int n = 1;
    ;
    for (i = 0; i < y; ++i)
      n *= x;
    return (n);
  }

  public static void main(String[] args) {
    int n, e, a;
    Scanner sc = new Scanner(System.in);
    System.out.println("Enter the Number : \n");
    n = sc.nextInt();
    System.out.println("Enter the Exponent : \n");
    e = sc.nextInt();
    a = CalcPow(n, e);
    System.out.println("The final Answer is : \n" + a);
    sc.close();
  }
}
